kubernetes/rollup-chart/templates/configmap.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.rollupName }}-conductor-env
+  namespace: astria-dev-cluster
+data:
+  home_dir: "/home/conductor"
+  executor_host_grpc_port: "{{ .Values.ports.executionGRPC }}"
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.rollupName }}-conductor-scripts
+  namespace: astria-dev-cluster
+data:
+  start-conductor.sh: |
+    {{- .Files.Get "files/scripts/start-conductor.sh" | nindent 4 }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.rollupName }}-conductor-keys
+  namespace: astria-dev-cluster
+data:
+  libp2p.key: |
+    {{- .Files.Get "files/keys/libp2p.key" | nindent 4 }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.rollupName }}-geth-env
+  namespace: astria-dev-cluster
+data:
+  home_dir: "/home/geth"
+  COMETBFT_RPC_ENDPOINT: "http://sequencer-service:26657"
+  executor_host_http_port: "{{ .Values.ports.jsonRPC }}"
+  executor_host_grpc_port: "{{ .Values.ports.executionGRPC }}"
+  executor_host_ws_port: "{{ .Values.ports.wsRPC }}"
+  sequencer_host_grpc_port: "9100"
+  genesis_alloc_address: "{{ .Values.genesisAllocAddress }}"
+  chain_id: "{{ .Values.chainId }}"
```

Comment:
kinda confusing naming wise here, but can we call this "evm_chain_id", for geth submission and execution we will need a ENV var called `CHAIN_ID` ([source](https://github.com/astriaorg/go-ethereum/blob/astria/internal/ethapi/sequencer.go#L19))

Reply:
@joroshiba will `CHAIN_ID` also be equal to `Values.evmChainId`?

Reply:
No these two should be seperate, one of them is the string, should also be passed into conductor as the "chain-id" this is what is used for sequencing, the other is the evm chain id.

they probably could also be the same, but one is a string and the other a number.

Reply:
Oh right, that's the one that's been "ethereum", gotcha